moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h

```diff
@@ -920,20 +920,6 @@ class MOVEIT_PLANNING_SCENE_EXPORT PlanningScene : public std::enable_shared_fro
   /** \brief Outputs debug information about the planning scene contents */
   void printKnownObjects(std::ostream& out = std::cout) const;
 
-  /** \brief Check if a message includes any information about a planning scene, or it is just a default, empty message.
-   */
-  [[deprecated("Use moveit/utils/message_checks.h instead")]] static bool
-  isEmpty(const moveit_msgs::msg::PlanningScene& msg);
-
-  /** \brief Check if a message includes any information about a planning scene world, or it is just a default, empty
-   * message. */
-  [[deprecated("Use moveit/utils/message_checks.h instead")]] static bool
-  isEmpty(const moveit_msgs::msg::PlanningSceneWorld& msg);
-
-  /** \brief Check if a message includes any information about a robot state, or it is just a default, empty message. */
-  [[deprecated("Use moveit/utils/message_checks.h instead")]] static bool
-  isEmpty(const moveit_msgs::msg::RobotState& msg);
```

Comment:
It looks like you still need to delete the definition of these functions in `planning_scene/src/planning_scene.cpp`